src/mpc-hc/CDarkTheme.cpp

```diff
@@ -267,12 +286,45 @@ CSize CDarkTheme::GetTextSizeDiff(CString str, HDC hDC, int type, CFont * curFon
     return cs - curCs;
 }
 
+
+void CDarkTheme::drawScaledImage(HDC hdc, CRect rect, UINT resID) {
+    Gdiplus::Graphics graphics(hdc);
+    CPngImage pngimage;
+    pngimage.Load(resID, AfxGetInstanceHandle());
+    Gdiplus::Bitmap image(pngimage, NULL);
+    //Gdiplus::Bitmap image(GetModuleHandle(NULL), MAKEINTRESOURCE(resID));
+    if (image.GetLastStatus() != 0)
+        return;
+    
+    graphics.SetInterpolationMode(Gdiplus::InterpolationModeHighQualityBicubic);
+    graphics.SetInterpolationMode(Gdiplus::InterpolationModeNearestNeighbor);

```

Comment:
One of these should be removed?

Reply:
I'm not using this function at the moment, but I was experimenting with resizing assets at various DPIs.  The second line is for testing a different algorithm.  They both look terrible, so I've decided to create custom assets for all the DPIs, but it might come in handy for other situations, so I left the code for now.